plugins/org.jboss.tools.ws.jaxrs.ui/plugin.xml

```diff
@@ -636,5 +636,16 @@
       </perspectiveExtension>
    </extension>
 
-       
+   <extension
+         point="org.eclipse.ui.ide.projectConfigurators">
+      <projectConfigurator
+            class="org.jboss.tools.ws.jaxrs.ui.internal.importer.JaxRsConfigurator"
+            label="Java EE Jax-RS">
+         <activeWhen>
+            <hasFileRecursively
+                  filename="web.xml">

```

Comment:
Yes, it means that this import wouldn't work with projects without a web.xml.
Do you know how m2e detects jax-rs in case there is no web.xml? Only by looking at the dependencies? /cc @fbricon